index.js

```diff
@@ -260,12 +262,55 @@
         const resultContent = `<h4 style="color: ${color}">${text}</h4>`
         addModalContent(resultContent)
       }
+      bindButtonEvents(button, onClick)
       document.body.append(button)
     } catch (e) {
       console.error(e.message)
     }
   }
 
+  const bindButtonEvents = (button, onClick) => {
+    let x = 0
+    let y = 0
+    let drag = false
+    const mouseDownHandler = function (e) {
+      // Get the current mouse position
+      x = e.clientX
+      y = e.clientY
+
+      // Attach the listeners to `document`
+      document.addEventListener('mousemove', mouseMoveHandler)
+      document.addEventListener('mouseup', mouseUpHandler)
+    }
+
+    const mouseMoveHandler = function (e) {
+      drag = true
+      // How far the mouse has been moved
+      const dx = e.clientX - x
+      const dy = e.clientY - y
+
+      // Set the position of element
+      button.style.top = `${button.offsetTop + dy}px`
```

Comment:
could the position be stored in local storage and fetched after restart?

Reply:
discussed with Veronika - not needed for now. Lets wait for feedback

Reply:
done